core/src/test/scala/org/apache/spark/sql/delta/DeltaSourceSuite.scala

```diff
@@ -337,6 +337,68 @@ class DeltaSourceSuite extends DeltaSourceSuiteBase with DeltaSQLCommandTest {
     }
   }
 
+  test("maxFilesPerTrigger: Trigger.AvailableNow respects read limits") {
+    withTempDir { inputDir =>
+      val deltaLog = DeltaLog.forTable(spark, new Path(inputDir.toURI))
```

Comment:
nit: We have a `File` version so that you can use `DeltaLog.forTable(spark, inputDir)`

Reply:
Gotcha, I just did what all the other tests were doing but can start simplifying these